tests/expected_results/end-to-end/reg_log_7a

```diff
@@ -0,0 +1,11 @@
+INFO:2020-02-05 17:29:01,552:AtacWorks-calculate-metrics] Loading intervals
```

Comment:
can we rename this file to be more descriptive of the content?

Reply:
done.